reference/ams-configuration.md

```diff
@@ -30,22 +30,46 @@ AMS provides various configuration items to customize its behavior. The followin
 
 Some features can be conditionally enabled via AMS.
 
+All feature flags can either be globally configured for all containers in the AMS configuration
+
+    amc config set container.features foo,bar
+
+or per application in the manifest
+
+    name: my-app
+    instance-type: a4.3
+    features: ["foo", "bar"]
+
 #### System UI
 
 *since 1.10.2*
 
-By default Anbox hides the Android system UI when an application is running in foreground mode. In some use cases it's however required to have the system UI available for navigation purposes. The system UI can be enabled via feature flag, e.g.:
-
-    $ amc config set container.features "enable_system_ui"
+By default Anbox hides the Android system UI when an application is running in foreground mode. In some use cases it's however required to have the system UI available for navigation purposes. This can be enabled with the `enable_system_ui` feature flag.
```

Comment:
```suggestion
By default, Anbox hides the Android system UI when an application is running in foreground mode. In some use cases, however, it's required to have the system UI available for navigation purposes. This can be enabled with the `enable_system_ui` feature flag.
```